blog/2015-02-26-Getting-Started-With_Docker-And-Wildfly.adoc

```diff
@@ -0,0 +1,195 @@
+
+ docker run -it -p 8080:8080 -p 9990:9990 jboss/wildfly
+
+In this way, we can now open our browser to `http://localhost:8080` to see the Wildfly welcome page. 
+
+=== Mapping a deployment folder
+
+Since we're not exposing SSH in this example, and we're not using the Management profiles in JBossTools, 
+we need a way for our server to pick up our deployments. The easiest way to do this is to map in a local folder 
+from our host into our container. For this example, we'll make a temporary directory somewhere on our host, 
+and tell our Docker container to treat that as the standalone/deployments folder inside the container. In this way, 
+changes the container makes to the folder can be visible on the host, and vice versa. 
+
+*Mapping Folders may cause IO errors for SELinux!*  To ensure your container can actually read and write to the folder, 
+you'll need to run `setenforce 0` to disable SELinux, or, alternatively, give Docker the permissions and exceptions in SELinux. 
+
+....
+[root@localhost jbds_wf_external]$  mkdir /home/rob/tmp/dockertest1
+[root@localhost jbds_wf_external]$  docker run -it -p 8080:8080 -p 9990:9990 \
+-v /home/rob/tmp/dockertest1:/opt/jboss/wildfly/standalone/deployments/:rw  jboss/wildfly
+....
+
+If you were to now place a .war file inside `/home/rob/tmp/dockertest1`, it will be picked up by the deployment scanner, 
+and visible in a web browser. 
+
+=== Making your Server Adapter
+
+The final step of this example is to create your Wildfly 8.2 server adapter in JBossTools, 
+and to create and deploy a web application to your temporary folder, which in my case is
+`/home/rob/tmp/dockertest1`
+
+First, we'll open the `Servers View` and create a new `Wildfly 8.x` server adapter.
+Since we're exposing our container's ports on our localhost, we can leave the host as 
+`localhost`. Make sure your server is marked as `Local` and Controlled by `Filesystem and shell operations` 
+as shown below. 
+
+image::images/20150226_docker/local_fs_1.png[]
+
+Since we're hacking a Local Filesystem server adapter to work for Docker, we'll still need a local
+runtime in this example, so there's no real way to avoid that. 
+
+=== Configuring your Server Adapter
+
+Once your server is created, you'll find it in the `Servers View`, where we can double-click
+it to open the `Server Editor`.  From here, we can make what configuration changes we'll need. 
+First, we'll need to make sure the server is `Externally Managed`.  This means 
+our attempts to start and stop it in Eclipse will be mostly ignored, since we're
+expecting the user to handle that via Docker. 
+
+image::images/20150226_docker/local_fs_externally_managed.png[]
+
+Next, we'll need to disable the tooling for keeping deployment scanners in sync with
+the locations JBossTools expects to be deploying. Since we've already mapped the folder
+in via the Docker command line, we won't need any additions to the deployment scanners at all. 
```

Comment:
It will attempt to add scanners via mgmt. I believe this is the last case where Local+fs uses mgmt at all, and thats bc there's no easy way to remove this usecase aside from unchecking the option in server editor.